### PR TITLE
[PIN-3447] Integrated support role to authorization process

### PR DESCRIPTION
**Pull request by rGregnanin**

### CODE REVIEW

**Review by galales — CHANGES_REQUESTED** [review]
missing `getClientOperatorRelationshipById`

**Comment by galales** [comment]
> missing `getClientOperatorRelationshipById`

nvm, it has recently been removed

**Review by galales — APPROVED** [review]

**Comment by beetlecrunch** [comment]
LGTM, I noticed that the authorization test for the `getClientsWithKeys` function is completely missing.
Could you add it (src/test/resources/authz.json)?
Thank you!

File: src/test/resources/authz.json [hunk 1]
```diff
@@ -33,12 +33,17 @@
     {
       "route": "getClientKeyById",
       "verb": "GET",
-      "roles": ["admin", "security", "m2m"]
+      "roles": ["admin", "security", "m2m", "support"]
+    },
+    {
+      "route": "getClientsWithKeys",
```
Comment (TonioGela) [inline src/test/resources/authz.json:39]: Please pay attention, as that's not enough, you should add the test that uses this value, exactly like in other cases too.

Comment (rGregnanin) [reply src/test/resources/authz.json:39]: Like the other PR i apologize for the misunderstanding, next time I pay more attention

**Review by TonioGela — APPROVED** [review]

**Pull request merged**